src/moaicore/MOAIFreeTypeFont.cpp

```diff
@@ -343,10 +343,10 @@ int MOAIFreeTypeFont::_setDefaultSize(lua_State *L){
 FT_Face MOAIFreeTypeFont::AffirmFreeTypeFace () {
 	
 	if (!this->mFreeTypeFace) {
-		FT_Library library;
-		FT_Error error = FT_Init_FreeType( &library );
+		this->mLibrary = new FT_Library();
+		FT_Error error = FT_Init_FreeType( this->mLibrary );
 		CHECK_ERROR(error);
-		return this->LoadFreeTypeFace( &library );
+		return this->LoadFreeTypeFace( this->mLibrary );
```

Comment:
Nope, this is the only place where `mLibrary` can get assigned, and the `if (!this->mFreeTypeFace) {` check at the beginning of `AffirmFreeTypeFace` guards against the creation of multiple library objects. :guardsman:  There is exactly one library object instance per `MOAIFreeTypeFont` instance.  :books: :one: